src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -271,6 +271,11 @@ public void OnStopActivity(Activity activity, object payload)
 
             if (TryFetchResponse(payload, out HttpResponseMessage response))
             {
+                if (currentStatusCode == ActivityStatusCode.Unset)
+                {
+                    activity.SetStatus(SpanHelper.ResolveSpanStatusForHttpStatusCode(activity.Kind, (int)response.StatusCode));
```

Comment:
is this change related to "error.type" ?

Reply:
No, this was there already https://github.com/open-telemetry/opentelemetry-dotnet/blob/53458639086f5a3afc39c97b21ca63b25d61e64a/src/OpenTelemetry.Instrumentation.Http/Implementation/HttpHandlerDiagnosticListener.cs#L284-L287. I just moved it to the top.